apisix/plugins/log-rotate.lua

```diff
@@ -265,6 +265,22 @@ local function rotate_file(files, now_time, max_kept)
     end
 end
 
+local function get_custom_logfile_name()
+    local custom_file_name = core.config.local_conf()
```

Comment:
Is there a special function to determine the log file name, access.log or other custom names, and then delete DEFAULT_ACCESS_LOG_FILENAME？

Reply:
All I could find is the `local_conf()` func in `config_local.lua` but nothing specifically for getting custom names